src/multiselect_controls.js

```diff
@@ -162,8 +162,6 @@ export class MultiselectControls {
 
     // Original settings for handling the start block
     this.origHandleBlockStart = Blockly.Gesture.prototype.handleBlockStart;
-
-    this.origSetStartBlock = Blockly.Gesture.prototype.setStartBlock;
```

Comment:
Why remove this? Now this causes an error

![011ebc4e857efdd1c75ce1711f3ec4ab](https://github.com/user-attachments/assets/7639c428-1d02-443c-977e-a7a81996c613)

Reply:
Oh, I forgot to remove the line that adds back the gestuer in disable multiselect.

Reply:
I actually highly recommend having a restore of the original Blockly.Gesture.prototype.setStartBlock just in case, unless you have confirmed with the Blockly team that it's useless for sure